director/scripts/test/kinematic_test.py

```diff
@@ -30,21 +34,36 @@ def __init__(self, parent = None):
     def handle_thread(self):
         self.channel = "NETWORK_CASSIE_STATE_DISPATCHER"
         self.lcm = lcm.LCM()
+        self.prev_loc = {}
+        self.json_file = sys.argv[3];
```

Comment:
This use of `sys.argv` seems like it requires this particular script to be in a specific place in the full call to this method. I suspect there's a better way to do this.

I did some (incomplete) digging, but it looks like the argument should be available in this `globalsDict` object
https://github.com/RobotLocomotion/director/blob/7139b80d3d2902a7ffdb8e3eac731c6acc68aa92/src/python/director/mainwindowapp.py#L336

I don't know how you can access it, but the name implies it's global somehow.